posthog-node/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.5.3 - 2023-02-21
+
+1. Allow passing in a distinctId to `groupIdentify()`.
+2. Fix a bug with active feature flags on capture events, where non-active flags would be added to the list as well.
```

Comment:
can we add a lil test for this too? 🥺 For consistency across libraries and test spec documentation

Reply:
I did, that's what the 'disabled_flag: false' is doing in tests

Reply:
(so I effectively added two tests 😂 )

Reply:
oops didn't see originally it but i guess that works LOL :P